callisto/notification/api.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import gnupg
+import pytz
+
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.utils import timezone
+
+from callisto.delivery.models import SentMatchReport
+from callisto.notification.models import EmailNotification
+
+logger = logging.getLogger(__name__)
+tzname = settings.REPORT_TIME_ZONE or 'America/Los_Angeles'
+timezone.activate(pytz.timezone(tzname))
+
+
+class NotificationApi(object):
```

Comment:
how about a `NotificationApi` in `callisto/delivery/api.py` that looks like this

```python
class NotifcationApi(object):
    def __init__(self, override_class):
        # the override_class arg is for testing / quick overrides
        if not override_class:
            override_class = get_class_from_string( getattr(
                settings,
                'CALLISTO_NOTIFICATION_API',
                'callisto.notification.api.NotificationApi',
            ) )
        self.api = override_class
```

and a `NotificationApi` in `callisto/notification/api.py` that looks like this

```python
class Notification(object):
    def send_email_to_person(self):
        ...
```

so you get calls in `callisto/delivery/views.py` like this

```python
from callisto.delivery.api import NotificationApi

NotifcationApi().api.send_email_to_person()
```